config/initializers/service_providers.rb

```diff
@@ -0,0 +1,3 @@
+SERVICE_PROVIDERS = YAML.load_file("#{Rails.root}/config/service_providers.yml")
```

Comment:
I'm wondering why do this here rather than at the ServiceProviderConfig class level.   e.g.. why no just have `@@service_proveriders ||=` in the initialiser?   Seems like it would be nicer since it would keep all the code together.   I guess the good thing about static inits is maybe that they push the startup cost to startup time and errors show at startup?   However I'm not sure the cost here is high enough to warrant a special initializer file?